[2022-08-29 15:27:32,564] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: h1b_lca_data_v3.entry_task scheduled__2021-04-10T06:00:00+00:00 [queued]>
[2022-08-29 15:27:32,584] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: h1b_lca_data_v3.entry_task scheduled__2021-04-10T06:00:00+00:00 [queued]>
[2022-08-29 15:27:32,586] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-08-29 15:27:32,588] {taskinstance.py:1239} INFO - Starting attempt 14 of 14
[2022-08-29 15:27:32,590] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-08-29 15:27:32,612] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): entry_task> on 2021-04-10 06:00:00+00:00
[2022-08-29 15:27:32,624] {standard_task_runner.py:52} INFO - Started process 11113 to run task
[2022-08-29 15:27:32,630] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'h1b_lca_data_v3', 'entry_task', 'scheduled__2021-04-10T06:00:00+00:00', '--job-id', '1509', '--raw', '--subdir', 'DAGS_FOLDER/upload_h1b_data_to_gcp.py', '--cfg-path', '/tmp/tmpszfv641h', '--error-file', '/tmp/tmp534_r4oi']
[2022-08-29 15:27:32,632] {standard_task_runner.py:77} INFO - Job 1509: Subtask entry_task
[2022-08-29 15:27:32,727] {logging_mixin.py:109} INFO - Running <TaskInstance: h1b_lca_data_v3.entry_task scheduled__2021-04-10T06:00:00+00:00 [running]> on host efb24f1b01c6
[2022-08-29 15:27:32,845] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=h1b_lca_data_v3
AIRFLOW_CTX_TASK_ID=entry_task
AIRFLOW_CTX_EXECUTION_DATE=2021-04-10T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-04-10T06:00:00+00:00
[2022-08-29 15:27:32,850] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-29 15:27:32,852] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo 04']
[2022-08-29 15:27:32,879] {subprocess.py:85} INFO - Output:
[2022-08-29 15:27:32,888] {subprocess.py:89} INFO - 04
[2022-08-29 15:27:32,892] {subprocess.py:93} INFO - Command exited with return code 0
[2022-08-29 15:27:32,933] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=h1b_lca_data_v3, task_id=entry_task, execution_date=20210410T060000, start_date=20220829T152732, end_date=20220829T152732
[2022-08-29 15:27:32,967] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-08-29 15:27:33,015] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
